src/transport/SecureSessionMgr.cpp

```diff
@@ -202,11 +202,16 @@ CHIP_ERROR SecureSessionMgr::SendMessage(SecureSessionHandle session, PayloadHea
         ChipLogProgress(Inet, "Sending secure msg on connection specific transport");
         err = state->GetTransport()->SendMessage(state->GetPeerAddress(), std::move(msgBuf));
     }
-    else
+    else if (mTransportMgr != nullptr)
```

Comment:
This is a workaround for https://github.com/project-chip/connectedhomeip/issues/7297 I assume?  If so, make a note there and add a comment here?

Reply:
Seems like a good null check? I'm fine keeping them..